doc/manifest/schema/1.0.0/installer.md

```diff
@@ -0,0 +1,444 @@
+
+
+## Fields
+### PackageIdentifier
+<details>
+ <summary>The package unique identifier</summary>
+
+ #### Required Field
+
+ This key is the unique identifier for a given package. This value is generally in the form of `Publisher.Package`. It is case sensitve, and this value must match the folder structure under the partition directory in GitHub.
+
+</details>
+
+### PackageVersion
+<details>
+ <summary>The package version</summary>
+
+ #### Required Field
+
+ This key represents the version of the package. It is related to the specific release this manifests targets. In some cases you will see a perfectly formed [semantic](https://semver.org) version number, and in other cases you might see something different. These may be date driven, or they might have other characters with some package specific meaning for example.
+
+ The Windows Package Manager client uses this version to determine if an upgrade for a package is available. In some cases, packages may be released with a marketing driven version, and that causes trouble with the `winget upgrade` command. 
+
+ The current best practice is to use the value reported in Add / Remove Programs when this version of the package is installed. In some cases, packages do not report a version resulting in an upgrade loop or other unwanted behavior.
+</details>
+
+### Channel
+<details>
+ <summary>The distribution channel</summary>
+
+ #### Optional Field
+
+ This key represents the distribution channel for a package. Examples may include "stable" or "beta".
+
+ >Note: This key is included for future use. The Windows Package Manager currently does not have any behavior associated with this key. The intent behind this key is to help disambiguate the different channels for packages lacking support for side by side installation. Some packages support having more than one package channel available on a system simultaneously. This key is intended to ensure the proper channel for a package is used during install and upgrade scenarios.
+ 
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+
+</details>
+
+### Platform
+<details>
+ <summary>The installer supported operating system</summary>
+
+ #### Optional Field
+
+ This key represents the Windows platform targeted by the installer. The Windows Package Manager currently supports "Windows.Desktop" and "Windows.Universal".
```

Comment:
I thought this field is more for the Store source to check applicability. Currently winget client does not use this field